dist/index.html

```diff
@@ -4,8 +4,10 @@
         <meta charset="utf-8" />
         <title>Radiance</title>
         <link rel="stylesheet" href="./style.css"/ >
+        <script src="./ui.js"></script>
```

Comment:
I had to do this to get the "Hello, radiance" to show up (briefly):

```diff
diff --git a/dist/index.html b/dist/index.html
index adb5fe7..31a21c9 100644
--- a/dist/index.html
+++ b/dist/index.html
@@ -4,10 +4,10 @@
         <meta charset="utf-8" />
         <title>Radiance</title>
         <link rel="stylesheet" href="./style.css"/ >
-        <script src="./ui.js"></script>
+        <script src="./radianceui.js"></script>
     </head>
     <body>
-        <!--<script src="./radiance.js"></script>-->
         <radiance-graph></radiance-graph>
+        <script src="./libradiance.js"></script>
     </body>
 </html>
```

Reply:
I'm happy to merge it as-is though; I just wasn't sure if you wanted to change anything in `statc/index.html` or not?